relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -490,6 +534,42 @@ fn timestamp_by_op(spans: &[Annotated<Span>], op: &str) -> Option<Timestamp> {
         .and_then(|span| span.timestamp.value().copied())
 }
 
+/// Counts the number of dynamically loaded images from debug meta.
+fn count_images(debug_meta: &Annotated<DebugMeta>) -> Option<f64> {
+    debug_meta
+        .value()
+        .and_then(|debug_meta| debug_meta.images.value())
+        .map(|images| images.len() as f64)
+}
+
+/// Sum the sizes of dynamically loaded images from debug meta.
+fn sum_images_size(debug_meta: &Annotated<DebugMeta>) -> Option<f64> {
+    debug_meta
+        .value()
+        .and_then(|debug_meta| debug_meta.images.value())
+        .map(|images| {
+            images
+                .iter()
+                .filter_map(|image| {
+                    image
+                        .value()
+                        .and_then(get_image_size)
+                        .and_then(|size| size.value().map(|v| *v as f64))
+                })
+                .sum()
+        })
+}
+
+fn get_image_size(debug_image: &DebugImage) -> Option<Annotated<u64>> {
+    match debug_image {
+        DebugImage::Apple(image) => Some(image.image_size.clone()),
+        DebugImage::Symbolic(image) => Some(image.image_size.clone()),
+        DebugImage::MachO(image) => Some(image.image_size.clone()),
+        // TODO(nar): Add support for Android image types
```

Comment:
@jjbayer I noticed a few of these `DebugImage` types had `image_size` as properties whereas others like `Proguard` only had `other`. Do you know if in the cases of these "other" debug images, the image size could be stored in `other`?

I'm just wondering if you're familiar with this interface and if there's an easy way to optionally return the image size using a predictable access pattern like `image.image_size` or `image.other.image_size`.

I'm also waiting for word back from the SDKs about which of these images types are actually applicable to my use case so this list may grow/shrink, e.g. I don't know which types are typically sent for Android other than Proguard yet